### DISCUSSION TITLE
Menu bar mistakenly disappeared

### DISCUSSION THREAD

**Discussion by fab442** [transmission-remote-gui/transgui#1464] [General]
Hi,
I am a Mac user of Transmission GUI. I unticked the "menu" from the "view" dropdown menu of the app on my Mac. How can I reset this ? 

<img width="1512" alt="Capture d’écran 2023-11-09 à 11 48 43" src="https://github.com/transmission-remote-gui/transgui/assets/78967210/af3547a7-8cc9-41de-895b-0b477a2de1c7">

thanks for this app!

**Comment by lighterowl** [comment]
https://github.com/xavery/transgui/issues/72#issuecomment-1848947261
